plugins/org/src/components/Cards/OwnershipCard/OwnershipCard.test.tsx

```diff
@@ -263,10 +263,11 @@ describe('OwnershipCard', () => {
       },
     );
 
-    expect(getByText('OPENAPI').closest('a')).toHaveAttribute(
-      'href',
-      '/create/?filters%5Bkind%5D=api&filters%5Btype%5D=openapi&filters%5Bowners%5D=my-team&filters%5Buser%5D=all',
-    );
+    const href = getByText('OPENAPI').closest('a')?.href ?? '';
```

Comment:
This test was initially failing, so refactored to get a better view of what was breaking. Both old and new version now passes, but will keep the refactor.